registration/registration.go

```diff
@@ -168,6 +177,51 @@ func (r *Registration) CertifierInfo() *CertifierConfig {
 	return r.cfg.Certifier
 }
 
+func (r *Registration) LoadTrustedPublicKeys() error {
+	if r.cfg.Certifier == nil {
+		return ErrCertificationIsNotSupported
+	}
+
+	loadedKeys := make([][]byte, 0)
+
+	dirPath := r.cfg.Certifier.TrustedKeysDirPath
+	if dirPath == "" {
+		return ErrTrustedKeyDirPathIsNotSet
+	}
+
+	err := filepath.Walk(dirPath, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if info.IsDir() || filepath.Ext(path) != ".key" {
+			return nil
+		}
+
+		keyData, err := os.ReadFile(path)
+		if err != nil {
+			return err
+		}
+
+		key := Base64Enc{}
+		if err = key.UnmarshalFlag(string(keyData)); err != nil {
+			return err
+		}
+
+		if len(key.Bytes()) != ed25519.PublicKeySize {
+			return ErrInvalidPublicKey
+		}
+
+		loadedKeys = append(loadedKeys, key.Bytes())
+		return nil
```

Comment:
`Base64Enc` exists to allow to pass base64 encoded command line parameters to PoET. Reading base64 encoded keys from a file can be done with standard library tools only:

```suggestion
		f, err := os.Open(path)
		if err != nil {
			return err
		}

		dec := base64.NewDecoder(base64.StdEncoding, f)
		key, err := io.ReadAll(dec)
		if err != nil {
			return ErrInvalidPublicKey
		}
		if len(key) != ed25519.PublicKeySize {
			return ErrInvalidPublicKey
		}
		loadedKeys = append(loadedKeys, key)
		return nil
```

This also allocates memory for the key only once (when reading it from the file) while the previous approach allocates memory for the same key 2 times: once for the string representation and once for the raw bytes.